src/Tasks/ResolveAssemblyReferences/Server/ServerMutex.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Threading;
+
+namespace Microsoft.Build.Tasks.ResolveAssemblyReferences.Server
+{
+    internal sealed class ServerMutex : IDisposable
+    {
+        private readonly Mutex _mutex;
+        public bool IsLocked { get; private set; }
+        public bool IsDisposed { get; private set; }
+
+        public ServerMutex(string name, out bool createdNew)
```

Comment:
an out in a .ctor is unfamiliar syntax.  Is this a must-have?

Reply:
Fixed.